backend/schemas/README.md

```diff
@@ -6,9 +6,11 @@ Follow these steps to version bump the workbook templates:
 - Run `make all` to generate new schemas and tempaltes
 - Once your PR is merged, don't forget to copy the new templates, found in `backend/schemas/output/excel/xlsx/`, into `assets/workbooks/` of the [static site repo](https://github.com/GSA-TTS/FAC-transition-site).
 
-# Updating the CFDA listings
+# Updating the ALN (formerly CFDA) listings
 
-The current CFDA assistance listings are in the CSV found [here](https://sam.gov/data-services/Assistance%20Listings/datagov?privacy=Public). When downloading the file, save it in the format `cfda-lookup-YYYYMMDD.csv` in the `/schemas/source/data` directory. Running `make all` should be sufficent to regenerate the lookup schemas and the Excel templates.
+The current CFDA assistance listings are in the CSV found [here](https://sam.gov/data-services/Assistance%20Listings/datagov?privacy=Public). 
```

Comment:
```suggestion
```

I'd suggest removing this bit since the new process doesn't rely on sam.gov's data exports anymore.